packages/rum-core/src/domain/rumEventsCollection/view/viewCollection.spec.ts

```diff
@@ -50,7 +54,7 @@ describe('viewCollection', () => {
         loadEvent: 10,
       },
     }
-    lifeCycle.notify(LifeCycleEventType.VIEW_UPDATED, view as View)
+    lifeCycle.notify(LifeCycleEventType.VIEW_UPDATED, (view as any) as View)
```

Comment:
We tend to limit as much as possible the usages of any
Could we do something better?

Reply:
I fixed this by casting the `location` attribute to `Partial<Location>`

Reply:
This actually didn't work so I ended up using `window.location` to have the proper type.